esp/scm/ws_workunits.ecm

```diff
@@ -1881,6 +1881,9 @@ ESPresponse [exceptions_inline] WUGetZAPInfoResponse
     string ThorIPAddress;
     string BuildVersion;
     string Archive;
+    [min_ver("1.73")] string EmailTo;
```

Comment:
Why are these returned? Who needs them? I can see that a UI may want to display the emailTo value but not the server/port

Reply:
Ok, I will remove the server/port.